### ISSUE TITLE
[Segment] Colored Segment Should Work in Segment Group

### ISSUE THREAD

**Issue by tubia** [Semantic-Org/Semantic-UI#4013]
Hi,
I'm trying to understand if it's possible to have a group of segments with one or more of them coloured.
Something like this:

``` html
<div class="ui segments">
    <div class="ui red segment"></div>
    <div class="ui segment"></div>
    <div class="ui segment"></div>
</div>
```

Currently, this rule override the border-top colour:

``` css
.ui.segments:not(.horizontal) > .segment:first-child {
border-top: none;
//...
}
```

Is this the expected result?

Thank you!

**Comment by jlukic** [comment]
I know there might be issues here, but this issue is not specific enough for me to address them concretely.

**Comment by kaurinz** [comment]
There seems to be some functional overlap between groups of segments, and attached segments. When the example above is implemented as a group of attached segments, it functions as expected.  

Not sure if it would work for all use cases, but using attached segments might be an appropriate workaround.

**Comment by jlukic** [comment]
Fixed in `2.2.11`
